website/templates/project/edit_draft_registration.mako

```diff
@@ -40,6 +40,7 @@
                       <ul class="list-group" data-bind="foreach: {data: Object.keys(page.questions), as: 'qid'}">
                         <span data-bind="with: page.questions[qid]">
                           <li data-bind="css: {
+                                           list-item-warning: $root.currentQuestion().hasErrors(),
```

Comment:
@barbour-em is currentQuestion correct? The question relevant to the knockout foreach context is available via ```$data```. 
http://knockoutjs.com/documentation/with-binding.html